hrm-domain/hrm-service/scripts/admin-commands/renotify.ts

```diff
@@ -0,0 +1,164 @@
+ */
+
+import type { HrmAccountId } from '@tech-matters/types';
+import type { ManuallyTriggeredNotificationOperation } from '@tech-matters/hrm-types';
+import { getSsmParameter } from '@tech-matters/ssm-cache';
+import { getHRMInternalEndpointAccess } from '@tech-matters/service-discovery';
+import { getAdminV0URL, staticKeyPattern } from '../hrmInternalConfig';
+// import envRegionMap from '../../../../.github/workflows/config/environment-region-map.json';
+
+const getAccountSid = ({
+  environment,
+  shortCode,
+}: {
+  environment: string;
+  shortCode: string;
+}) => {
+  return getSsmParameter(`/${environment}/twilio/${shortCode.toUpperCase()}/account_sid`);
+};
+
+const requestRenotify = async ({
+  accountSid,
+  authKey,
+  dateFrom,
+  dateTo,
+  entityType,
+  internalResourcesUrl,
+  operation,
+}: {
+  operation: ManuallyTriggeredNotificationOperation;
+  entityType: 'contacts' | 'cases' | 'profiles';
+  internalResourcesUrl: URL;
+  accountSid: HrmAccountId;
+  authKey: string;
+  dateFrom: string;
+  dateTo: string;
+}) => {
+  const url = getAdminV0URL(
+    internalResourcesUrl,
+    accountSid,
+    `/${entityType}/${operation}`,
+  );
+
+  console.info(`Submitting request to ${url}`);
+
+  const response = await fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `Basic ${authKey}`,
+    },
+    body: JSON.stringify({ dateFrom, dateTo }),
+  });
+
+  if (!response.ok) {
+    console.error(
+      `Failed to submit request for ${operation} ${entityType}: ${response.statusText}`,
+    );
+    console.info(await response.text());
+  } else {
+    console.info(
+      `Performing ${operation} ${entityType} from ${dateFrom} to ${dateTo}...`,
+    );
+    console.info(await response.text());
+  }
+};
+
+export const renotify = async ({
+  region,
+  environment,
+  accounts,
+  dateFrom,
+  dateTo,
+  contacts,
+  cases,
+  profiles,
+  operation,
+}: {
+  region: string;
+  environment: string;
+  accounts: string[];
+  dateFrom: string;
+  dateTo: string;
+  contacts: boolean;
+  cases: boolean;
+  profiles: boolean;
+  operation: ManuallyTriggeredNotificationOperation;
+}) => {
+  try {
+    // TODO: access all regions (once eu-west-1 is accessible via management vpn)
+    // console.log('envRegionMap', envRegionMap);
+    const timestamp = new Date().getTime();
+    const assumeRoleParams = {
+      RoleArn: 'arn:aws:iam::712893914485:role/tf-admin',
+      RoleSessionName: `hrm-admin-cli-${timestamp}`,
+    };
+
+    const { authKey, internalResourcesUrl } = await getHRMInternalEndpointAccess({
+      region,
+      environment,
+      staticKeyPattern,
+      assumeRoleParams,
+    });
+
+    for (const account of accounts) {
+      const accountSid = (await getAccountSid({
+        environment,
+        shortCode: account,
+      })) as HrmAccountId;
+
+      console.log(`Processing ${operation} request for account ${accountSid}`);
+      if (contacts) {
```

Comment:
```
[contacts ? 'contacts' : null, cases ? 'cases' : null, profiles ? 'profiles' : null].filter(Boolean).forEach(...)
```

Instead of the duplication maybe?